src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainPhoto = document.querySelector('.gallery__large-img');
+const galleryList = document.querySelector('.gallery__list');
+
+function clickOnGallery(event) {
+  const listItem = event.target.closest('.list-item');
+
+  if (listItem) {
+    event.preventDefault();
+
+    const listLink = listItem.querySelector('.list-item__link');
+    const photoItem = listLink.querySelector('.gallery__thumb');
+    const itemSource = photoItem.getAttribute('src');
+
+    mainPhoto.setAttribute('src', itemSource);
```

Comment:
Currently, the main image source is being set to the thumbnail image source. If the intention is to display the full-size image when a thumbnail is clicked, you should use the 'href' attribute of the '.list-item__link' element instead of the 'src' attribute of the '.gallery__thumb' element.